charts/pre-api/values.yaml

```diff
@@ -42,3 +44,4 @@ java:
     AZURE_INGEST_SA: "preingestsa{{ .Values.global.environment }}"
     AZURE_FINAL_SA: "prefinalsa{{ .Values.global.environment }}"
     PLATFORM_ENV_TAG: Production
+    ISSUER: https://sts.windows.net/531ff96d-0ae9-462a-8d2d-bec7c0b42082/
```

Comment:
I assume this is a tenant level thing so consistent across all envs?

Reply:
Yeah I believe so- at least seems to be the same value in stg, test and demo envs.